pkg/kademlia/routing.go

```diff
@@ -147,6 +147,39 @@ func (rt *RoutingTable) GetBucketIds() (storage.Keys, error) {
 	return kbuckets, nil
 }
 
+// DumpNodes loops returns all nodes in the nodeBucketDB and marshals them to &pb.Nodes
+func (rt *RoutingTable) DumpNodes() ([]*pb.Node, error) {
```

Comment:
Jen is working on replacing all the calls to List() w/ iterators [https://github.com/storj/storj/pull/1318].  I'd use routing.go's iterate() https://github.com/storj/storj/blob/36e4d8c1c28594cb9b30950db227cc7fd025886e/pkg/kademlia/routing.go#L262 here